subtitles.go

```diff
@@ -313,6 +386,7 @@ func (s *Subtitles) Add(d time.Duration) {
 	for idx := 0; idx < len(s.Items); idx++ {
 		s.Items[idx].EndAt += d
 		s.Items[idx].StartAt += d
+
```

Comment:
I'm being picky here, but can you not add this line feed? It messes up the git diff

Reply:
Done